script/start

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -e
+
+# colours
+YELLOW='\033[1;33m'
+NC='\033[0m' # no colour - reset console colour
+
+IGNORE_IF_ALREADY_RUNNING=false
+
+while test $# -gt 0; do
+  case "$1" in
+    -i|--ignore-if-already-running)
+      IGNORE_IF_ALREADY_RUNNING=true
```

Comment:
The name of this variable leads me to think that if set to true this script will ignore the command if nginx if already running, but below on line 25 is doing the opposite. Is that right?

Reply:
Getting back to this. Couldn't re-open this PR for some reason, so https://github.com/guardian/dev-nginx/pull/22. Feedback welcome.

Reply:
Probably because the `master` branch does not exist anymore?

Reply:
> Probably because the `master` branch does not exist anymore?

Ah good spot! Surprised it's not possible to change the base branch, but 🤷🏽 .

Reply:
You should be able to `git rebase main`